projects/admin/src/app/service/record-routing.service.ts

```diff
@@ -206,7 +206,11 @@ export class RecordRoutingService {
             canRead: (record: any) => this.canReadItem(record),
             canUpdate: (record: any) => this.canUpdate(record),
             canDelete: (record: any) => this.canDelete(record),
-            preprocessRecordEditor: (record: any) => this.preprocessItem(record)
+            preprocessRecordEditor: (record: any) => this.preprocessItem(record),
+            redirectUrl: (record: any) => {
+              const pidRegExp = new RegExp('.*\/(.*)$');
+              return of('records/documents/detail/' + record.document.$ref.match(pidRegExp)[1]);
```

Comment:
Use back tick here. 
And it would be nice if you test the return of the regex, before using it.